src/app/modules/auth/classic-auth/classic-auth.service.ts

```diff
@@ -198,6 +224,8 @@ export class ClassicAuthService {
       name: existingClassicCredentials.name
     });
 
+    await this.usersService.activate(existingClassicCredentials.user_id);
```

Comment:
here you must add transaction because you update 2 entities: classic_auth and user